mobile/Screen/EventDetailScreen.js

```diff
@@ -1,48 +1,244 @@
+    marginTop: 80,
+    borderTopLeftRadius: 25,
+    borderTopRightRadius: 25,
   },
-  h1: {
-    fontSize: 30,
-    fontWeight: 'bold',
+  overlayContent: {
+    flex: 1,
+    marginTop: 100,
+    marginHorizontal: 10,
   },
-  h2: {
+  title: {
+    width: '80%',
+    fontFamily: 'MontserratMedium',
+    fontSize: 24,
+  },
+  header: {
+    marginTop: 40,
+    marginBottom: 20,
+    fontFamily: 'MontserratMedium',
     fontSize: 20,
-    fontWeight: '30',
   },
-  square: {
-    width: 40,
-    height: 40,
-    backgroundColor: '#D9D9D9',
-    margin: 40,
+  organizationContainer: {
+    flexDirection: 'row',
+    backgroundColor: '#F2F2F6',
+    borderRadius: 50,
+    marginTop: 10,
+    marginBottom: 35,
+    alignItems: 'center',
   },
-  image: {
-    width: 200,
-    height: 200,
+  organizationText: {
+    fontFamily: 'MontserratMedium',
+    fontSize: 16,
+    color: '#424288',
+    marginHorizontal: 10,
+    marginVertical: 4,
   },
-  heading: {
+  organizationInfoIcon: {
+    width: 20,
+    height: 20,
+  },
+  info: {
+    backgroundColor: '#EFEFF5',
+    borderWidth: 0.5,
+    borderColor: '#D0D0E8',
+    borderRadius: 5,
+    padding: 20,
+  },
+  infoIcon: {
+    width: 30,
+    height: 30,
+    marginRight: 15,
+    resizeMode: 'contain',
+  },
+  infoIconTransparent: {
+    marginRight: 15,
+    width: 20,
+    height: 20,
+    resizeMode: 'contain',
+  },
+  saveIcon: {
+    width: 23,
+    height: 23,
+    resizeMode: 'contain',
+  },
+  arrowIcon: {
+    width: 10,
+    height: 13,
+    resizeMode: 'contain',
+    marginLeft: 15,
+  },
+  readMore: {
+    fontFamily: 'MontserratSemiBold',
+    fontSize: 14,
+    lineHeight: 20.3,
+    color: '#1E2021',
+  },
+  infoText: {
+    fontFamily: 'MontserratSemiBold',
+    fontSize: 16,
+    color: '#4C4C9B',
+  },
+  infoImage: {
+    width: 20,
+    height: 20,
+  },
+  infoRow: {
     flexDirection: 'row',
     alignItems: 'center',
+    marginBottom: 20,
+    width: '90%',
+  },
+  lastRow: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  frequency: {
+    backgroundColor: '#D0D0E8',
+    borderRadius: 60,
+    paddingVertical: 5,
+    paddingHorizontal: 9,
+    marginLeft: 15,
+  },
+  frequencyText: {
+    fontFamily: 'MontserratMedium',
+    fontSize: 12,
+    color: '#4C4C9B',
+  },
+  heading: {
+    fontFamily: 'MontserratMedium',
+    fontSize: 20,
+    marginBottom: 20,
+    marginTop: 10,
+  },
+  description: {
+    fontFamily: 'Montserrat',
+    fontSize: 13,
+    lineHeight: 20,
   },
 });
 
 function EventDetailScreen({
   navigation, route,
 }) {
   const {
-    id, title, organizations, day, location, time, summary, url, number, email, website,
-  } = route.params;
+    id,
+    image,
+    startTime,
+    endTime,
+    day,
+    week,
+    location,
+    title,
+    description,
+    startDate,
+    organization,
+    recurringMonthly,
+    recurringWeekly,
+  } = route.params || {};
+
+  const organizations = [
```

Comment:
I'm guessing this is hard-coded test data that, for the actual app, we'll be pulling from Mongo DB? Maybe can you move see test data objects into mongoDB so that we can ensure that we're pulling data properly, and so that we're not putting hardcoded events in here?